src/components/Header.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <router-link
+    to="/"
+    class="flex py-6 bg-white items-center"
+    v-if="!is_embedded"
```

Comment:
I will need to play around with this but I guess this means if the route is embedded then don't show the header